contracts/GSN/GSNContext.sol

```diff
@@ -19,7 +19,13 @@ contract GSNContext is Initializable, Context {
     event RelayHubChanged(address indexed oldRelayHub, address indexed newRelayHub);
 
     function initialize() public initializer {
-        _upgradeRelayHub(0xD216153c06E857cD7f72665E0aF1d7D82172F494);
+        setDefaultRelayHub();
+    }
+
+    function setDefaultRelayHub() public {
+        if (_getRelayHub() == address(0)) {
```

Comment:
Would you consider reverting when `_getRelayHub()` returns a non-zero address (i.e. if it already has been initialized)? This would make it consistent with `initializer` behavior.

Reply:
The current `initializer` behavior is actually against our own guidelines which is for initializers to be idempotent. I think it would be nicer for `setDefaultRelayHub` to revert, though. Are you OK with revert in `setDefault` + no revert in the constructor?

Reply:
Yes!